kube/services/jupyterhub/jupyterhub-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: jupyterhub-deployment
+spec:
+  replicas: 1
```

Comment:
Remove replicas.
Default is 1, and we can scale up via `patch_kreps` function in kubes.sh, and not worry about re-applying a deployment.yaml